client.go

```diff
@@ -953,7 +953,7 @@ func (f *File) WriteTo(w io.Writer) (int64, error) {
 
 		if inFlight == 0 {
 			if firstErr.err == nil && len(pendingWrites) > 0 {
-				return copied, InternalInconsistency
+				return copied, ErrInternalInconsistency
```

Comment:
If you make it an alias above, then this should be OK.

Reply:
done